client/src/Components/Alerts/AlertList.js

```diff
@@ -1,15 +1,17 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-import { Redirect, useHistory} from 'react-router-dom';
+import { Redirect, useHistory } from 'react-router-dom';
 
 import webHost from '../../config/web';
 import api from '../../config/web';
 import CreateAlert from './CreateAlert';
+import { PROFILE } from "../../constants/constants";
 
 const AlertList = () => {
   const [list, setList] = useState([]);
   const [loading, setLoading] = useState(true);
   const history = useHistory();
+  const creatorId = JSON.parse(localStorage.getItem(PROFILE)).result._id;
```

Comment:
is this value `localStorage.getItem(PROFILE)` always guaranteed to exist? If not, what should we do to handle scenarios where it doesn't exist?